### DISCUSSION TITLE
question about multiplicative weights

### DISCUSSION THREAD

**Discussion by katecycho** [open2c/cooler#397] [Q&A] [answered]
Hello,

I am trying to generate a bias file for FitHiC2 from my .mcool file. 
I was wondering how I can extract the multiplicative weights from the cool files? I could only find this clr.matrix(divisive_weights=True)[:] but wasn't sure how to dump the weights. Thank you so much for the help.

**Comment by nvictus** [comment, marked as answer]
If the weights are divisive (like "KR"), you would simply need to grab them from the bin table and invert them:

```python
multiplicative_weights = 1/clr.bins()["KR"][:]
```
